### PR TITLE
Document relates to include for timetracking.list and timetracking.info

### PR DESCRIPTION
**Pull request by gehadMohamed**
add `relates_to` include to timetracking.list and timetracking.info endpoints docs

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -9621,3 +9629,12 @@ Fetch which version of Projects the account is using.
...
 + `sunday`
+
+## TimeTrackingRelatesToTypes (enum)
++ `contact`
++ `company`
++ `project`
++ `milestone`
++ `ticket`
++ `nextgenProject`
++ `nextgenGroup`
```
Comment (BrechtBonte) [inline apiary.apib]: Q: I don't think we actually support time trackings related to nextgen groups, unless I missed something
/cc @adognameddog

Comment (gehadMohamed) [reply apiary.apib]: There is an open PR with the changes to include it

**Review by joaoinacio — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (adognameddog) [reply apiary.apib:9640]: What do you mean @gehadMohamed? Allowing time tracking on project groups is a massive feature. Can you point me to that open PR?

Comment (joaoinacio) [inline apiary.apib]:
```suggestion
+ `nextgenProjectGroup`
```

**Review by JordyDeGroeve — DISMISSED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by StephenBeirlaen — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Pull request merged**